tests/unit/schema/wrappers/test_field.py

```diff
@@ -44,6 +44,14 @@ def test_field_is_primitive():
     assert primitive_field.is_primitive
 
 
+def test_field_deprecated():
+    field = make_field('DeprecatedField', is_deprecated=True)
+    assert field.is_deprecated
+
+    field = make_field('DeprecatedField', is_deprecated=False)
```

Comment:
Maybe use a different field name (at least the case when `is_deprecated=False`).

(Same comment for message and method below.)

Reply:
Fixed in 5b6edb9